packages/app-project/src/helpers/fetchWorkflowsHelper/fetchWorkflowsHelper.js

```diff
@@ -67,12 +70,12 @@ async function buildWorkflow(workflow, displayName, env) {
     id: workflow.id,
     links: workflow.links,
     prioritized: workflow.prioritized,
-    subjectSets: []
```

Comment:
The add-on subjectSets property is handled in each page file for /classify/workflow/[workflowId] and not needed in the helper function, which is intended to be agnostic to whether a workflow is configured for subject set selection.

You'll see this remove from spec files too.